website/lib/update-color-examples.tsx

```diff
@@ -0,0 +1,215 @@
+import css from "css";
+import { readFileSync } from "fs";
+import { noCdnClient } from ".";
+import dotenv from "dotenv";
+
+dotenv.config();
+
+type SemanticColorEntry = {
+  title: string;
+  color_value: string;
+  color_name: string;
+  category: string;
+  subcategory: string;
+};
+
+type GlobalColorEntry = {
+  title: string;
+  color_value: string;
+  category: string;
+  subcategory: string;
+};
+
+function declarationToColor(declaration, declarationList): [string, string] {
+  var value: string;
+  var name: string;
+  if (declaration.value.startsWith("var(")) {
+    const target = declaration.value.slice(4, -1);
+    const resolved = declarationList.find((d) => d.property == target);
+    if (resolved === null) {
+      return null;
+    }
+    value = declaration.value;
+    name = resolved.property.replace("--navds-global-color-", "");
+  } else {
+    value = declaration.value;
+  }
+  return [name, value];
+}
+
+function parseDeclaration(
+  declaration,
+  declarationList
+): SemanticColorEntry | GlobalColorEntry {
+  const property = declaration.property;
+  const value = declaration.value;
+  const ignoredTags = ["navds"];
+  const globalAndSemanticTags = ["semantic", "global"];
+  const specialCases = ["text", "divider", "focus", "border"];
```

Comment:
Tror `link` må legges til her da den ikke vil være `text-link` når tokens blir oppdatert snart. Vil dette fungere for nå, og når den er oppdatert?
```suggestion
  const specialCases = ["text", "divider", "focus", "border", "link"];
```

Reply:
ja, jeg tror det vil funke uansett ja. smart!